[2020-05-21 00:25:32,263][INFO ][cluster.metadata         ] [Sugar Man] [haystack] creating index, cause [api], templates [], shards [5]/[1], mappings []
[2020-05-21 00:25:33,263][INFO ][cluster.metadata         ] [Sugar Man] [haystack] create_mapping [modelresult]
[2020-05-21 00:25:33,457][INFO ][cluster.routing.allocation] [Sugar Man] Cluster health status changed from [RED] to [YELLOW] (reason: [shards started [[haystack][2], [haystack][2], [haystack][4]] ...]).
[2020-05-21 00:25:33,496][INFO ][cluster.metadata         ] [Sugar Man] [haystack] update_mapping [modelresult]
[2020-05-21 00:25:34,947][WARN ][http.netty               ] [Sugar Man] Caught exception while handling client http traffic, closing connection [id: 0xea193344, /127.0.0.1:61502 => /127.0.0.1:9200]
java.io.IOException: Удаленный хост принудительно разорвал существующее подключение
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
[2020-05-21 00:27:15,202][INFO ][cluster.metadata         ] [Sugar Man] [haystack] creating index, cause [api], templates [], shards [5]/[1], mappings []
[2020-05-21 00:27:15,735][INFO ][cluster.routing.allocation] [Sugar Man] Cluster health status changed from [RED] to [YELLOW] (reason: [shards started [[haystack][0], [haystack][0]] ...]).
[2020-05-21 00:27:15,760][INFO ][cluster.metadata         ] [Sugar Man] [haystack] create_mapping [modelresult]
[2020-05-21 00:27:15,870][INFO ][cluster.metadata         ] [Sugar Man] [haystack] update_mapping [modelresult]
[2020-05-21 00:27:16,492][WARN ][http.netty               ] [Sugar Man] Caught exception while handling client http traffic, closing connection [id: 0xda0baf61, /127.0.0.1:61610 => /127.0.0.1:9200]
java.io.IOException: Удаленный хост принудительно разорвал существующее подключение
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
[2020-05-21 00:35:52,812][WARN ][http.netty               ] [Sugar Man] Caught exception while handling client http traffic, closing connection [id: 0x62b94406, /127.0.0.1:61908 => /127.0.0.1:9200]
java.io.IOException: Удаленный хост принудительно разорвал существующее подключение
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
[2020-05-21 00:36:07,451][INFO ][cluster.metadata         ] [Sugar Man] [haystack] creating index, cause [api], templates [], shards [5]/[1], mappings []
[2020-05-21 00:36:07,595][INFO ][cluster.metadata         ] [Sugar Man] [haystack] create_mapping [modelresult]
[2020-05-21 00:36:07,698][INFO ][cluster.routing.allocation] [Sugar Man] Cluster health status changed from [RED] to [YELLOW] (reason: [shards started [[haystack][4]] ...]).
[2020-05-21 00:36:07,720][INFO ][cluster.metadata         ] [Sugar Man] [haystack] update_mapping [modelresult]
[2020-05-21 00:36:08,064][WARN ][http.netty               ] [Sugar Man] Caught exception while handling client http traffic, closing connection [id: 0x7e57181b, /127.0.0.1:62035 => /127.0.0.1:9200]
java.io.IOException: Удаленный хост принудительно разорвал существующее подключение
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
[2020-05-21 00:36:48,526][WARN ][http.netty               ] [Sugar Man] Caught exception while handling client http traffic, closing connection [id: 0xf392c45b, /127.0.0.1:62081 => /127.0.0.1:9200]
java.io.IOException: Удаленный хост принудительно разорвал существующее подключение
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
[2020-05-21 00:37:00,400][INFO ][bootstrap                ] running graceful exit on windows
[2020-05-21 00:37:00,401][INFO ][node                     ] [Sugar Man] stopping ...
[2020-05-21 00:37:00,572][INFO ][node                     ] [Sugar Man] stopped
[2020-05-21 00:37:00,573][INFO ][node                     ] [Sugar Man] closing ...
[2020-05-21 00:37:00,581][INFO ][node                     ] [Sugar Man] closed
